dgp/lib/etc.py

```diff
@@ -7,6 +7,130 @@
 import numpy as np
 
 
+def align_frames(frame1, frame2, align_to='left', interp_method='time',
+                 interp_only=[], fill={}):
+    # TODO: Is there a more appropriate place for this function?
+    # TODO: Add ability to specify interpolation method per column.
+    # TODO: Ensure that dtypes are preserved unless interpolated.
+    """
+    Align and crop two objects
+
+    Parameters
+    ----------
+    frame1: :obj:`DataFrame` or :obj:`Series
+        Must have a time-like index
+    frame1: :obj:`DataFrame` or :obj:`Series
```

Comment:
Duplicate parameter docstring here to fix later.

Reply:
Oops. Thanks for the heads-up.